adam-cli/src/test/scala/FlagStatTest.scala

```diff
@@ -0,0 +1,136 @@
+package org.bdgenomics.adam.cli
+import org.apache.spark.rdd.RDD
+
+import org.bdgenomics.adam.projections.{AlignmentRecordField, Projection}
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.read.{DuplicateMetrics, FlagStatMetrics}
+import org.bdgenomics.adam.util.SparkFunSuite
+import org.bdgenomics.formats.avro.AlignmentRecord
+import org.bdgenomics.adam.rdd.read.FlagStat._
+
+class flagstat_test extends SparkFunSuite with java.io.Serializable   {
+
+  sparkTest("Standard FlagStat test"){
+
+    val inputpath = ClassLoader.getSystemClassLoader.getResource("features/NA12878.adam").getFile
```

Comment:
Looks like they meant to add it to the commit but forgot it. Good catch @massie!

Also @beaunorgeot, when you add it, can you add NA12878.adam as a SAM file? We try to keep the test resources as text files instead of binary files where possible (I believe there are a few small exceptions, but we try to limit those), as git handles text files better, and it also makes debugging tests a bit easier (you have the text and don't need to poke into a binary file).